src/jaguar.toit

```diff
@@ -447,11 +466,18 @@ serve_incoming_requests socket/tcp.ServerSocket id/uuid.Uuid name/string address
       writer.write
           json.encode registry_.entries
 
+    // Handle installing containers.
+    else if path == "/install" and request.method == "PUT":
+      container_name ::= headers.single HEADER_CONTAINER_NAME
+      defines ::= extract_defines headers
+      install_image request.content_length request.body container_name defines
+      writer.write
+          json.encode {"status": "OK"}
```

Comment:
Could be interesting to have an `OK_` static, but maybe not worth it.

Reply:
Will look at that in another PR.